nomad/structs/variables.go

```diff
@@ -35,10 +38,41 @@ const (
 	// Reply: VariablesByNameResponse
 	VariablesReadRPCMethod = "Variables.Read"
 
+	// VariablesRenewLockRPCMethod is the RPC method for renewing the lease on
+	// a lock according to its namespace, path and lock ID.
+	//
+	// Args: VariablesRenewLockRequest
+	// Reply: VariablesRenewLockResponse
+	VariablesRenewLockRPCMethod = "Variables.RenewLock"
+
 	// maxVariableSize is the maximum size of the unencrypted contents of a
 	// variable. This size is deliberately set low and is not configurable, to
 	// discourage DoS'ing the cluster
 	maxVariableSize = 65536
+
+	// minLockTTL and maxLockTTL determine the range of valid durations for the
+	// TTL on a lock.They come from the experience on Consul.
+	minLockTTL = 10 * time.Second
```

Comment:
Nitpick: these new constants and error variables are visible at the package level and aren't going to be "obviously wrong" by name. If they were named things like `minVariableLockTTL` then it'll be obvious if accidentally misused for another purpose in this package (`structs` is a very large package).

Reply:
Got it, let's rename them then